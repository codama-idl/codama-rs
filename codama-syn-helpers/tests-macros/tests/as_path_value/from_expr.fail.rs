use codama_syn_helpers_test_macros::as_path_value;

#[as_path_value("hello")]
pub struct Test;

fn main() {}
