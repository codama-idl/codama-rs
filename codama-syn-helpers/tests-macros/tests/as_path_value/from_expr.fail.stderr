error: expected a path followed by a value: `my_path = ...`
 --> tests/as_path_value/from_expr.fail.rs:3:17
  |
3 | #[as_path_value("hello")]
  |                 ^^^^^^^
